code/src/java/pcgen/gui3/util/MappedList.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * MappedList.java
+ * Copyright 2016 Connor Petty <cpmeister@users.sourceforge.net>
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * 
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+ * 
+ * Created on Aug 27, 2016, 1:22:00 PM
+ */
+package pcgen.gui3.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+import javafx.collections.ListChangeListener;
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.TransformationList;
+
+/**
+ *
+ * @author Connor Petty <cpmeister@users.sourceforge.net>
+ */
+public class MappedList<E, F> extends TransformationList<E, F> {
```

Comment:
nit: no @autdhor
please provide some text for what the class does in the comments.